packages/corvid-e2e-tests/test/drivers/cliDriver.js

```diff
@@ -33,3 +34,48 @@ module.exports = ({ cwd }) => {
     openEditor
   };
 };
+
+module.exports = (...args) => {
+  const commands = createCommandsDrvier(args);
+
+  const isLoggedIn = async () => {
+    const loginDebugPort = await findFreePort();
+    const loginCommand = commands.login({
+      remoteDebuggingPort: loginDebugPort
+    });
+    const loginEditorDriver = await connectToLocalEditor(loginDebugPort);
+    console.log("login connected"); //eslint-disable-line
+
+    try {
+      await loginEditorDriver.waitForLoginForm();
+      await loginEditorDriver.close();
+      return false;
+    } catch (e) {
+      await loginCommand;
+      return true;
+    }
+  };
+
+  const performLogout = async () => await commands.logout();
+
+  const performLogin = async () => {
+    const loginDebugPort = await findFreePort();
+    const loginCommand = commands.login({
+      remoteDebuggingPort: loginDebugPort
+    });
+
+    const loginEditorDriver = await connectToLocalEditor(loginDebugPort);
+    console.log("login connected"); //eslint-disable-line
+
+    await loginEditorDriver.waitForLoginForm();
+    await loginEditorDriver.login(getCorvidTestUser());
+    await loginCommand;
+  };
+
+  return {
+    commands,
+    performLogout,
```

Comment:
i think performLogout & performLogin shouldn't be part of the driver. especially not if using the corvidTestUser.